src/Akeneo/Pim/Enrichment/Bundle/Command/MigrateToUuid/MigrateToUuidCommand.php

```diff
@@ -44,30 +48,38 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $withStats = $input->getOption('with-stats');
         $context = new Context($input->getOption('dry-run'), $withStats);
 
-        foreach ($this->steps as $stepIndex => $step) {
-            $output->writeln(sprintf('<info>Step %d: %s</info>', $stepIndex + 1, $step->getDescription()));
+        foreach ($this->steps as $step) {
+            $this->contextProcessor->push(['step' => $step->getName()]);
+            $this->logger->notice($step->getDescription());
```

Comment:
```suggestion
            $this->logger->info('Start step ' . $step->getDescription(), ['step' => $step->getName()]);
```

- I thought we said to use info and not notice :thinking: 
- Please don't inject monolog processor. We can inject context in 2nd argument of notice/info/warning/error methods
- I suggest to add 'Start step ' in front of the message in order to be able to facilitate the search in datadog